nncf/quantization/algorithms/weight_compression/openvino_backend.py

```diff
@@ -163,12 +163,11 @@ def transform_model(
                 converted_zero_point = opset.convert(zero_point_const, const_dtype)
                 converted_const = opset.subtract(converted_const, converted_zero_point)
 
-            scale_const = opset.constant(
-                compressed_weight.scale.data, dtype=const_dtype, name=f"{const_node_name}/scale"
-            )
+            scale_const = opset.constant(compressed_weight.scale.data, dtype="float16", name=f"{const_node_name}/scale")
+            converted_scale_const = opset.convert(scale_const, const_dtype)
```

Comment:
It looks like you should not insert convert operation if `const_dtype` is float16.

Reply:
Thanks, done